Tests/ChoreService_test.py

```diff
@@ -400,6 +400,22 @@ def test_exists(self):
         self.assertTrue(self.tm1.chores.exists(self.chore_name2))
         self.assertTrue(self.tm1.chores.exists(self.chore_name3))
         self.assertFalse(self.tm1.chores.exists(uuid.uuid4()))
+        
+    def test_search_for_process_name_no_match(self):
+        chore_names = self.tm1.chores.search_for_process_name(process_name="NotAProcessName")
+        self.assertEqual([], chore_names)
+
+    def test_search_for_process_name_happy_case(self):
+        chore_names = self.tm1.chores.search_for_process_name(process_name=process_name1)
+        self.assertEqual(self.chore_name1, chore_names[0].name)
+
+    def test_search_for_parameter_value_no_match(self):
+        chore_names = self.tm1.chores.search_for_parameter_value(parameter_value='NotAParamValue')
+        self.assertEqual([], chore_names)
+
+    def test_search_for_parameter_value_happy_case(self):
+        chore_names = self.tm1.chores.search_for_parameter_value(parameter_value='UK')
+        self.assertEqual(self.chore_name1, chore_names[0].name)        
```

Comment:
Same here. I think it would be good to assert that `len(chore_names) == 1`

Reply:
assertions updated with this commit 0c1c0d84bdc3bfd8532c6ba9898e41df5740ae9b